### ISSUE TITLE
unload detection

### ISSUE THREAD

**Issue by Commissar0617** [Courseplay/courseplay#1328]
i have the fliegel overloading trailer (http://www.farmingmods2015.com/fliegl-overload-station-v-1-2-trailer/), and courseplay doesn't seem to recognize that unload point for the tipper. it has an unload trigger, similar to the storage

**Comment by tjb8907** [comment]
I've tried this as well, along with another mod....the trigger isn't the same as storage, from what I can tell the Overloading Station is like a shovel trigger, while other ones are actual TipTriggers. 

http://www.farmingsimulatormods.net/farming-simulator-2015-mods/bunker-silo-v-1-0-sp-farming-simulator-2015-mod/

that is the one I have tried as well. I think the mod needs to have an actual Tip Trigger (area trigger with a start and end) integrated into it, which I believe would require a script as well. Maybe someone that knows a little more than myself will chime in. Unfortunately the develops aren't really doing much with courseplay anymore, so we might be on our own.

**Comment by Commissar0617** [comment]
huh... it seems to function the same, idk
 

    On Wednesday, June 8, 2016 2:22 PM, tjb8907 <notifications@github.com> wrote:
 

 I've tried this as well, along with another mod....the trigger isn't the same as storage, from what I can tell the Overloading Station is like a shovel trigger, while other ones are actual TipTriggers. http://www.farmingsimulatormods.net/farming-simulator-2015-mods/bunker-silo-v-1-0-sp-farming-simulator-2015-mod/that is the one I have tried as well. I think the mod needs to have an actual Tip Trigger (area trigger with a start and end) integrated into it, which I believe would require a script as well. Maybe someone that knows a little more than myself will chime in. Unfortunately the develops aren't really doing much with courseplay anymore, so we might be on our own.—
